### DISCUSSION TITLE
how to configure the advanced object search in pimcore 6.9.1

### DISCUSSION THREAD

**Discussion by Rojal123245** [pimcore/pimcore#8835] [Q&A / Support]
can someone help me in the configuration part of advanced object bundle. i have installed the bundle through composer but i dont know where to setup the index prefix in pimcore structure.
![image](https://user-images.githubusercontent.com/25191918/115348398-4f441c80-a1d2-11eb-876b-196a74d2721f.png)

**Comment by fashxp** [comment]
it is part of symfony configuration, so for example in the `app/config/config.yml` or any other yml configuration file, that is included in your container build.
